.github/workflows/validate.yml

```diff
@@ -90,3 +90,41 @@ jobs:
 
       - name: test
         run: make TESTFLAGS="${{ matrix.race }}" test
+
+  vm:
+    name: "VM"
+    strategy:
+      fail-fast: false
+      matrix:
+        template:
+          - template://almalinux-8
+          - template://centos-stream-9
+          - template://fedora
+    runs-on: ubuntu-24.04
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: "Install Lima"
+        uses: lima-vm/lima-actions/setup@v1
+        id: lima-actions-setup
+
+      - name: "Cache ~/.cache/lima"
+        uses: actions/cache@v4
+        with:
+          path: ~/.cache/lima
+          key: lima-${{ steps.lima-actions-setup.outputs.version }}
```

Comment:
Do I understand it right that this step caches and restores the image downloaded by `limactl start`?

If yes, the image name should probably be the key here, not the lima version. Or, if you foresee that the download format will change drastically soon, both the lima version (perhaps just a major version?) and the image name.

Reply:
One other observation, such caching can be internal to `lima-actions/setup`.

But that would not work for the case when there are different images. Hmm.

Reply:
Added the image name to the key

> Or, if you foresee that the download format will change drastically soon, both the lima version (perhaps just a major version?) and the image name.

It is not a matter of the format, it is a matter of the qcow2 digest in [`/usr/local/share/lima/templates/fedora.yaml`](https://github.com/lima-vm/lima/blob/v1.0.6/templates/fedora.yaml)